api/matlab/examples/Us4R_control.m

```diff
@@ -2,30 +2,31 @@
 % path to the MATLAB API files
 addpath('../arrus');
 
-txFrequency = 6e6;
+txFrequency = 2.5e6;
 samplingFrequency = 65e6;
 fsDivider = 1;
 
 [filtB,filtA] = butter(2,[0.5 1.5]*txFrequency/(samplingFrequency/fsDivider/2),'bandpass');
 
 %% Initialize the system, sequence, and reconstruction
-us	= Us4R('probeName',   'SL1543', ...
+us	= Us4R('probeName',   '2.25L32', ...
```

Comment:
This script should stay as before (i.e. without the interface, e.g. using SL1543 probe).

Reply:
OK